corsheaders/conf.py

```diff
@@ -1,65 +1,53 @@
 from django.conf import settings
 
 from .defaults import default_headers, default_methods  # Kept here for backwards compatibility
-from .validators import return_type
```

Comment:
Thank a lot. I already delete .validators, but it seems Django import errors. Do you have any advices?